src/Core/Scheduler/Providers/WMTS_Provider.js

```diff
@@ -20,19 +20,24 @@ function customUrl(layer, url, tilematrix, row, col) {
 
 function preprocessDataLayer(layer) {
     layer.fx = layer.fx || 0.0;
-
     layer.options = layer.options || {};
 
+    if (!layer.format && layer.options.mimetype) {
```

Comment:
Couldn't it be moved in `View` in `addLayer` or `preprocessLayer` ? To avoid duplicates.